flask_rebar/swagger_generation/generator_utils.py

```diff
@@ -282,3 +282,28 @@ def recursively_convert_dict_to_ordered_dict(obj):
         return [recursively_convert_dict_to_ordered_dict(item) for item in obj]
     else:
         return obj
+
+
+def get_parsed_method_docstring(method_definition, parse_docstring):
+    if method_definition.func.__doc__:
+        if parse_docstring:
+            try:
+                from docstring_parser import parse
```

Comment:
Can we import this at the top of the file like this 
```
try:
    from docstring_parser import parse
except ImportError:
    parser = None
```

And then do a check here for `if parser is not None`, This will prevent hitting this exception every time, and make this a little cleaner. Handling exceptions is also significantly slower than checking a boolean.

Reply:
updated